src/shared/Logo/ui/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { LOGO_URL } from "../lib/constants"
+
+export interface LogoProps {
+  widthPx?: number
+  heightPx?: number
+  classNameExtended?: string
+}
+
+const Logo = ({ widthPx, heightPx, classNameExtended }: LogoProps) => {
+  const defaultSize = {
+    height: 50,
+    width: 120,
+  }
+
+  const defaultClassName = "mr-1 p-1"
+
+  return (
+    <img
+      height={heightPx || defaultSize.height}
+      width={widthPx || defaultSize.width}
+      className={classNameExtended ? `${defaultClassName} ${classNameExtended}` : defaultClassName}
```

Comment:
Please consider using [classnames](https://www.npmjs.com/package/classnames) or any decent alternative if you intend to make such class mixins.

Reply:
Done